src/dragndrop.js

```diff
@@ -0,0 +1,32 @@
+export function eventlisteners(arr) {
+  arr.forEach((element) => {
+    const task = document.getElementById(element.index);
+    task.addEventListener('dragstart', (event) => {
+      event.dataTransfer.setData('index', element.index);
+    });
+    task.addEventListener('drop', (event) => {
```

Comment:
- [ ] Kindly ensure that dragging a todo item switches its position. Its position should get updated on dragend.

Reply:
- [ ] Also ensure that the position switching gets updated on localStorage.

Reply:
Sorry@mikenath223 but according to exercise requirements the user gets the correct list values after the page reloads and that's what it do

Reply:
> Sorry@mikenath223 but according to exercise requirements the user gets the correct list values after the page reloads and that's what it does

Reply:
> * [ ]  Also ensure that the position switching gets updated on localStorage.

Line 19 of dragndrop.js:
window.localStorage.setItem('tasklist', JSON.stringify(arr));

did you try refreshing the page after dragging and dropping?